v3/src/components/graph/adornments/plotted-value/plotted-value.tsx

```diff
@@ -145,7 +137,7 @@ export const PlottedValue = observer(function PlottedValue (props: IProps) {
   return (
     <>
       <div className="plotted-value-container" id={`plotted-value-${containerId}`}>
-        { model.value &&
+        { value && model.isVisible &&
```

Comment:
```suggestion
        { value != null && model.isVisible &&
```
Otherwise, zero values won't plot.

Reply:
Right. I'll have to do a better job of remembering that.